### PR TITLE
fix compile errors after merge

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: libraries/shared/src/ShapeInfo.cpp [hunk 1]
```diff
@@ -361,7 +360,7 @@ const DoubleHashKey& ShapeInfo::getHash() const {
             numHulls = 1;
         }
         if (numHulls > 0) {
-            hash = DoubleHashKey::hashFunction(numHulls, primeIndex++);
+            uint32_t hash = DoubleHashKey::hashFunction(numHulls, primeIndex++);
```
Comment (AndrewMeadows) [inline libraries/shared/src/ShapeInfo.cpp]: After merge `hash` was no longer known to this context, so we declare a new variable.

File: libraries/shared/src/ShapeInfo.cpp [hunk 2]
```diff
@@ -67,7 +67,6 @@ void ShapeInfo::setParams(ShapeType type, const glm::vec3& halfExtents, QString
             break;
         case SHAPE_TYPE_BOX:
         case SHAPE_TYPE_HULL:
-        case SHAPE_TYPE_SIMPLE_HULL:
```
Comment (ZappoMan) [inline libraries/shared/src/ShapeInfo.cpp]: did you mean to remove this?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4420/HighFidelity-Beta-PR11678-c14bbb76e63d5ae159ddf27afc16b7511fe52a3b.dmg

File: libraries/shared/src/ShapeInfo.cpp (hunk 2 above)
Comment (AndrewMeadows) [reply libraries/shared/src/ShapeInfo.cpp]: Yes.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4420/HighFidelity-Beta-PR11678-c14bbb76e63d5ae159ddf27afc16b7511fe52a3b.exe

**Pull request merged**